include/ADIOSTypes.h

```diff
@@ -37,6 +42,96 @@ enum class IOMode
     COLLECTIVE = 1
 };
 
-} // end namespace
+// Alias the fixed sized typed into the adios namespace to make sure we're
+// always using the right ones.
+using std::size_t;
+
+using std::int8_t;
+using std::uint8_t;
+using std::int16_t;
+using std::uint16_t;
+using std::int32_t;
+using std::uint32_t;
+using std::int64_t;
+using std::uint64_t;
+
+// Not sure if we're really use these ones but we'll round it out for
+// completion
+using real32_t = float;
+using real64_t = double;
+using complex32_t = std::complex<float>;
+using complex64_t = std::complex<double>;
+
+// Get a fixed width integer type from a size specification
+template <size_t Bytes, bool Signed>
```

Comment:
Can bool Signed = false be default?   unsigned is used for most raw data, signed only if numerical data requires.